### PR TITLE
Moved everything into a folder and added appeal command

### LINKED ISSUE #1: unclear naming
the naming is confusing:

https://github.com/Bot-detector/Bot-detector-Twitter/blob/80e109ce7b3be2740a5c995b6c5f7f905426da8d/main.py#L10

### LINKED ISSUE #2: Feature: 🔮Prediction🔮
Currently created and enabled. At the moment checks the database for a prediction and returns only the most-likely prediction. Does not give a response if the check failed through the pipeline, but does give a response if the RSN is not valid. This can be modified.

### LINKED ISSUE #3: Feature: 🔨Ban Checking🔨
At the moment the ban-checking feature is enabled for players to @osrsbotdetector banstatus <rsn>. This allows players to see if the account is banned, or not, on the hiscores and our other pathway. This can be modified to be reduced, and only allow a limited amount per user per time period in order to maintain our proxies. Though at the moment it does seem to have benefits in terms of marketing/traction, so I think it would be okay to maintain, considering that normal users may also be able to use this in our discord.

### LINKED ISSUE #9: Store past 100 seen tweet IDs
This will prevent us from hitting our Twitter API limit, by not running code on tweets that have been seen before.

### PR DESCRIPTION
**Pull request by Ferrariic**

### CODE REVIEW

**Comment by extreme4all** [comment]
api seems a weird foldername for this project

**Comment by Ferrariic** [comment]
> api seems a weird foldername for this project

That's fair, what should it be named?

File: twitter_bot/Commands/appeal.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def appeal(text_removed_at, parent_tweet_id, api):
+    check = 'appeal'
+    if text_removed_at[:len(check)] == check:
+        response = 'Appeal your ban here: http://jgx.game/Ban - Note: Our system does not have the amount and depth of data that Jagex has on hand, and therefore cannot be used as evidence to dispute rule-breaking behavior.'
+        try:
+            api.update_status(status=response, in_reply_to_status_id=parent_tweet_id, auto_populate_reply_metadata=True)
+        except Exception as e:
```
Comment (extreme4all) [inline twitter_bot/Commands/appeal.py]: try not to catch generic exceptions

File: twitter_bot/Commands/appeal.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+            print(f'Status update error. {e}')
```
Comment (extreme4all) [inline twitter_bot/Commands/appeal.py]: make use of logging repository instead of printing

**Review by extreme4all — CHANGES_REQUESTED** [review]

**Comment by extreme4all** [comment]
remove example-command.py

instead of having big if else blocks, make a negating if function

if not true:
    return something
    
avoid generic Exceptions
make use of logging library

type hinting

for bans just hit the api

seems like all the functions have something generic going on

can text_removed_at be renamed to text
rename api to twitter_api?
shouldn't twitter_api be a global variable

do we have to cycle through like a dict of our commands, how does discord handle this?
func = [v for dict_functions if k == command][0]
msg = func()

your functions should return the message to be sent, every function does this
def func(text, parent_tweet_id):
    api.update_status(
        status="a message",
        in_reply_to_status_id=parent_tweet_id, auto_populate_reply_metadata=True
    )

**Comment by Ferrariic** [comment]
All major instances should be resolved, will do exception catching in next branch

**Comment by Ferrariic** [comment]
Resolves #9 #3 #2

**Comment by extreme4all** [comment]
> Resolves #9 #3 #2

tagging is good, but i also linked the issue

**Comment by Ferrariic** [comment]
> > Resolves #9 #3 #2
> 
> tagging is good, but i also linked the issue

Thanks!

**Pull request merged**